plone/app/discussion/browser/javascripts/comments.js

```diff
@@ -201,7 +201,7 @@ require([  // jshint ignore:line
         /**********************************************************************
          * Delete a comment and its answers.
          **********************************************************************/
-        $('input[name="form.button.DeleteComment"]').on('click', function () {
+        $('input[name="form.button.DeleteCommentComment"]').on('click', function () {
```

Comment:
`DeleteCommentComment` Is this change intentional?
I cannot immediately spot why this is needed.
Also below I see a reference to `DeleteComment`

Reply:
This was indeed wrong. Thank you for pointing to!

Reply:
Oh sorry I did not notice this one was closed :/